src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -100,6 +100,16 @@ protected ActionResult execute() {
             data.emailToEdit.emailId = emailId;
         }
 
+        if (addressModeOn && groupModeOn) {
+            statusToAdmin = "Email queued for sending." + "<br/>" + "Receipient: " + addressReceiverListString
+                            + "<br/>" + "Group receiver's list " + groupReceiverListFileKey;
+            statusToUser.clear();
+
+            statusToUser.add(new StatusMessage("Email will be sent within an hour to "
```

Comment:
@xpdavid agreed, sounds like a good idea. On a sidenote, what do you think about having statusToAdmin changed from a String into an ArrayList<String>? Since in some methods like this once, multiple statuses to admin needs to be sent.